frontend/app/services/EmailService.scala

```diff
@@ -14,18 +17,32 @@ trait EmailService extends LazyLogging {
   val feedbackAddress: String
 
   val client = {
-    val c= new AmazonSimpleEmailServiceClient()
+    val c = new AmazonSimpleEmailServiceClient()
     c.setRegion(Region.getRegion(Regions.EU_WEST_1))
     c
   }
 
+  private val digest = MessageDigest.getInstance("MD5")
+
+  def md5(salt: String, input: String): Option[String] = {
```

Comment:
I'd be tempted to use Guava's Hash functions here - Guava is already in Membership-Frontend, and the methods look a bit prettier than having to do `try/catch`? 

```
HashFunction hf = Hashing.md5();
HashCode hc = hf.newHasher().putString("foo@bar.com", Charsets.UTF_8).hash();
```

https://github.com/google/guava/wiki/HashingExplained